src/StaticWebAssetsSdk/Sdk/Sdk.StaticWebAssets.StaticAssets.ProjectSystem.props

```diff
@@ -11,6 +11,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0">
 
+  <!-- Input and output configuration -->
+  <PropertyGroup>
+    <StaticWebAssetRootPath Condition="'$(StaticWebAssetRootPath)' == ''">wwwroot</StaticWebAssetRootPath>
+  </PropertyGroup>
+  <ItemGroup Condition="@(StaticWebAssetSourcePath->Count()) == 0">
+    <StaticWebAssetSourcePath Include="wwwroot" />
+  </ItemGroup>
+
```

Comment:
Do we actually need to make these input sources configurable? I would rather avoid this, as they are web specific. If people want to follow different conventions, that can be done in the project file directly.

Reply:
Intention described in the comment above..